content/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -466,36 +466,45 @@ export const ConceptPage: NextPage<Props> = ({
   const relatedConceptsGroupLabel = 'Related topics';
   const personOrAllFields =
     conceptResponse.type === 'Person' || themePagesAllFields;
-  const navLinks = [
-    sectionsData.by.images?.totalResults
-      ? {
-          text: `Images ${getThemeTabLabel('by', conceptResponse.type)}`,
-          url: `#images-${getThemeTabLabel('by', conceptResponse.type)}`,
-        }
-      : undefined,
-    sectionsData.about.images?.totalResults
-      ? {
-          text: `Images ${getThemeTabLabel('about', conceptResponse.type)}`,
-          url: `#images-${getThemeTabLabel('about', conceptResponse.type)}`,
-        }
-      : undefined,
-    sectionsData.in.images?.totalResults
-      ? {
-          text: `Images ${getThemeTabLabel('in', conceptResponse.type)}`,
-          url: `#images-${getThemeTabLabel('in', conceptResponse.type)}`,
-        }
-      : undefined,
-    { text: 'Works', url: '#works' },
-    frequentCollaborators?.length && personOrAllFields
-      ? { text: 'Frequent collaborators', url: '#frequent-collaborators' }
-      : undefined,
-    relatedTopics?.length && personOrAllFields
-      ? {
-          text: relatedConceptsGroupLabel,
-          url: `#${dasherize(relatedConceptsGroupLabel)}`,
-        }
-      : undefined,
-  ].filter(isNotUndefined);
+  const buildNavLinks = () => {
+    const links: Link[] = [];
+
+    // Add image sections
+    const imageSections = ['by', 'about', 'in'] as const;
```

Comment:
could you reuse `tabOrder` (or are they different?)

Reply:
Good shout